contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -2574,6 +2575,68 @@ tsql_stmt :
 				{ $$ = NULL; }
 		;
 
+tsql_CreatePartitionStmt:
+			CREATE OptTemp TABLE qualified_name '(' OptTableElementList ')'
```

Comment:
If we don't include optional nodes (like OptTemp) in the tsql_CreatePartitionStmt rule, the parser was encountering  shift-reduce conflict because it couldn't determine whether to shift (for tsql_CreatePartitionStmt) or reduce (for CreateStmt) when encountering the optional nodes due to similarity between the grammar rules.

When parser will encounter a statement starting with CREATE TABLE, it couldn't reliably decide whether to apply the CreateStmt rule or the tsql_CreatePartitionStmt rule because the latter didn't include the OptTemp clause.